studyplus-android-sdk/src/main/java/jp/studyplus/android/sdk/internal/auth/Preference.kt

```diff
@@ -0,0 +1,54 @@
+package jp.studyplus.android.sdk.internal.auth
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.util.Log
+import jp.studyplus.android.sdk.BuildConfig
+
+internal class Preference(context: Context) {
+
+    private val preferences = context.getSharedPreferences(prefName, Context.MODE_PRIVATE)
+
+    val isAuthenticated: Boolean
+        get() = accessToken.isNotEmpty()
+
+    val accessToken: String
+        get() = preferences.getString(KEY_ACCESS_TOKEN, "") ?: ""
+
+    fun update(data: Intent) {
+        val code = data.getStringExtra(EXTRA_SP_AUTH_RESULT_CODE)
+        val token = data.getStringExtra(EXTRA_SP_AUTH_ACCESS_TOKEN)
+        if (code.isNullOrEmpty() || token.isNullOrEmpty()) {
+            Log.e(
+                "StudyplusSDK",
+                "The data is invalid. If code is correct, please contact Studyplus Dev team.",
+            )
+            return
+        }
+
+        if (code != RESULT_CODE_AUTHENTICATED) {
+            // not authenticated
+            return
+        }
```

Comment:
I think the SDK should return a log also here.

e.g. 
```kotlin
Log.e(
    "StudyplusSDK",
    "Your app is not authenticated.",
)
```

Reply:
I added 6a61fd6 !

Reply:
Thank you for your rapid response!!